internal/chess-bot-executable/chess-bot-executable.go

```
@@ -0,0 +1,67 @@
+	executable *executable.Executable
+	logger     *logger.Logger
+	args       []string
+}
+
+func NewChessBotExecutable(stageHarness *test_case_harness.TestCaseHarness) *ChessBotExecutable {
+	b := &ChessBotExecutable{
+		executable: stageHarness.NewExecutable(),
+		logger:     stageHarness.Logger,
+	}
+
+	stageHarness.RegisterTeardownFunc(func() { b.Kill() })
```

golangci-lint (1.64.8)

Error return value of `b.Kill` is not checked

(errcheck)